projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -1903,6 +1944,7 @@ public static function parse_fields_from_content( $post_id ) {
 			$content      = str_ireplace( array( '<br />', ')</p>' ), '', $content[1] );
 			$fields_array = preg_replace( '/.*Array\s\( (.*)\)/msx', '$1', $content );
 
+			// TODO: some explanation on this regex could help
```

Comment:
I'm not sure this is related to the rest of the changes here, but since you asked:

It's parsing the form from the post content. Unfortunately it's a bit unreliable because the format we're using doesn't have clear delimiters. I've tried to make it as resilent as I could but there's still some edge cases. For example try entering the following inside a multi-line text field:

```
So this is your regular value

Something => This will now become a new key-value pair when you view feedback.
```

Somehow, the export actually gets around all these issues I've been able to find with it, so we'll need to dig into that and see if we need this method at all.